### PR TITLE
Feature/221

### PR DESCRIPTION
**Pull request by Kamal-Technogetic**

### CODE REVIEW

File: src/stories/Avatar.stories.tsx [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+import { AvatarProps } from '../interfaces/CommonInterface';
 
 export default {
     title: 'ReactComponentLibrary/Avatar',
     component: Avatar
-} as ComponentMeta<typeof Avatar>;
+} as Meta<typeof Avatar>;
 
-const Template: ComponentStory<typeof Avatar> = (args) => <Avatar {...args} />;
+const Template: StoryFn<AvatarProps> = (args: AvatarProps) => <Avatar {...args} />;
```
Comment (rajeshdeveloper99) [inline src/stories/Avatar.stories.tsx:11]: what is StoryFn

Comment (Kamal-Technogetic) [reply src/stories/Avatar.stories.tsx:11]: IN Storybook 6.0 onward "StoryFn" refers to a new way of defining stories introduced in newer versions of Storybook.

**Review by rajeshdeveloper99 — COMMENTED** [review]
check some comments

File: src/stories/Avatar.stories.tsx (hunk 1 above)
Comment (rajeshdeveloper99) [reply src/stories/Avatar.stories.tsx:11]: ok

**Pull request merged**